jupyter-book/deconvolution/deconv_covid_MuSiC_v0.ipynb

```diff
@@ -0,0 +1,2511 @@
+{
```

Comment:
<p>I'd state here that you're not only showing MuSiC because of its performance in the benchmarks (cite them again here!), but also because it's not a web based tools.</p><p> </p><p> </p><p>"In practice, our reference profile might not include more"</p><p>Not "our", but "the" </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/theislab/cross-modal-single-cell-best-practices/pull/93/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

Reply:
<p>Please add what you want.</p>